src/Server.php

```diff
@@ -3,8 +3,8 @@
 namespace League\Glide;
 
 use InvalidArgumentException;
-use League\Flysystem\FileExistsException;
-use League\Flysystem\FilesystemInterface;
+use League\Flysystem\FilesystemException as FilesystemV2Exception;
```

Comment:
Maybe it would be more clear to rename this to or `FlysystemException` or `FlysystemFilesystemException` since V2 doesn't have any meaning in Glide.

Reply:
I don't have strong preference here, yet it's just an alias and aliases are scoped per file. Thus consumers of the library still will have to catch `League\Flysystem\FilesystemException` and the alias `FilesystemV2Exception` won't be exposed anywhere. To me it was useful name as it states exactly where the exception comes from.

But still I don't have strong preference :)

Reply:
It was a nitpick so LGTM.